testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -568,6 +569,136 @@ public void testTrinoWritingDataWithObjectStorageLocationProvider(StorageFormat
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
 
+    private static final String TRINO_INSERTED_PARTITION_VALUES = "" +
+            "(1, 'with-hyphen')," +
+            "(2, 'with.dot')," +
+            "(3, 'with:colon')," +
+            "(4, 'with/slash')," +
+            "(5, 'with\\\\backslashes')," +
+            "(6, 'with\\backslash')," +
+            "(7, 'with=equal')," +
+            "(8, 'with?question')," +
+            "(9, 'with!exclamation')," +
+            "(10, 'with%percent')," +
+            "(11, 'with%%percents')," +
+            "(12, 'with space')";
+
+    private static final String SPARK_INSERTED_PARTITION_VALUES = "" +
+            "(1, 'with-hyphen')," +
+            "(2, 'with.dot')," +
+            "(3, 'with:colon')," +
+            "(4, 'with/slash')," +
+            "(5, 'with\\\\\\\\backslashes')," +
```

Comment:
I'm not sure exactly why I needed 8 backslashes here to get a literal with 2 slashes.

Reply:
2 backslashes in the value x2 per https://stackoverflow.com/questions/41765033/why-does-sparksql-require-two-literal-escape-backslashes-in-the-sql-query  x2 for java string literal = 8